db/sports.json

```diff
@@ -0,0 +1,40 @@
+{
+    "sports": [
+        
+        {
+            "name": "Fútbol",
+            "slug": "futbol",
+            "description": "Fútbol"
+        },
+        {
+            "name": "Tenis",
+            "slug": "tenis",
```

Comment:
rules?

Reply:
las necesitamos de entrada? que propiedades realmente necesitamos para los sports para lo que vamos a mostrar en esta instancia?

Reply:
bubeno si tiene pocas sports ya que tenga de to aunque como veas me daigual en realidad